nginx/ng/servers_config.sls

```diff
@@ -112,6 +112,11 @@ nginx_server_available_dir:
     - name: {{ server_curpath(server) }}
     - source: {{ source_path }}
     - template: jinja
+    - require:
+      - file: nginx_server_enabled_dir
+      - file: nginx_server_available_dir
+    - require_in:
+      - service: nginx_service
```

Comment:
@EvaSDK so since we are doing a reverse require here for `nginx_service` we have to enforce that the parent folders of the config files exists (Either if it is custom or if it is `sites-available` and `sites-enabled`). That's why I had to put `makedirs=True`.